src/ios/CDVUIInAppBrowser.m

```diff
@@ -863,6 +887,7 @@ - (void)viewDidLoad
 - (void)viewDidUnload
 {
     [self.webView loadHTMLString:nil baseURL:nil];
+    self.webView.delegate = nil;
```

Comment:
Note: this code nulls the webView delegate on closing the IAB, in order to resolve an existing bug in the current UIWebView iOS implementation:

I have an app with ~10k daily iOS users and I was logging 10-15 crashes per day in Crashlytics due to this. The Crashlytics crash dump is as follows:

	#0 Crashed: WebThread EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x0000000172df1210
	0	libobjc.A.dylib	objc_msgSend + 16
	1	UIKitCore	-[UIWebView webView:resource:canAuthenticateAgainstProtectionSpace:forDataSource:] + 92
	2 WebKitLegacy	CallResourceLoadDelegateReturningBoolean(bool, void (*)(), WebView*, objc_selector*, objc_object*, objc_object*, objc_object*) + 76
	3	WebCore	WebCore::ResourceLoader::canAuthenticateAgainstProtectionSpace(WebCore::ProtectionSpace const&) + 64
	4	WebCore	WebCore::ResourceLoader::canAuthenticateAgainstProtectionSpaceAsync(WebCore::ResourceHandle*, WebCore::ProtectionSpace const&, WTF::CompletionHandler<void (bool)>&&) + 32
	5	WebCore	WTF::Function<void ()>::CallableWrapper<-[WebCoreResourceHandleAsOperationQueueDelegate connection:canAuthenticateAgainstProtectionSpace:]::$_3>::call() + 136
	6	JavaScriptCore	WTF::dispatchFunctionsFromMainThread() + 308
	7	Foundation	__NSThreadPerformPerform + 336
	8	CoreFoundation	__CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 24
	9	CoreFoundation	__CFRunLoopDoSource0 + 88
	10	CoreFoundation	__CFRunLoopDoSources0 + 176
	11	CoreFoundation	__CFRunLoopRun + 1040
	12	CoreFoundation	CFRunLoopRunSpecific + 436
	13	WebCore	RunWebThread(void*) + 592
	14	libsystem_pthread.dylib	_pthread_body + 128
	15	libsystem_pthread.dylib	_pthread_start + 48
	16 libsystem_pthread.dylib	thread_start + 4


The current IAB implementation sets a UIWebView delegate but never unsets it, which leads to the webview thread referencing a non-existent UIWebView instance after the IAB window is closed, due to the still-present delegate reference.

This corresponds to the [Apple documentation for UIWebView delegate](https://developer.apple.com/documentation/uikit/uiwebview/1617937-delegate?language=objc):

 > Before releasing an instance of `UIWebView` for which you have set a delegate, you must first set its delegate property to `nil`. This can be done, for example, in your dealloc method.

This code solves the issue by nilling the delegate on the webview when closing the IAB.
I have released an app update containing my own fork of this plugin which contains this fix and I have now seen these crashes stop being logged in Crashlytics in the new version.

If you'd rather this fix but pulled out and submitted as a separate PR, let me know.

Reply:
Would be awesome if this was a separate PR - I think this could be related to some memory leak reports we were also getting?

Reply:
OK, I will create a separate issue and PR for it.

But should I also remove this code fix from this PR (since it will be added by the above PR anyway)?

Reply:
If it is not too much work, you can do that.
Otherwise, we just make sure to merge your new PR first, then this will just be already present when merging this one, thus being credited as a change of the other PR.

Reply:
redundant nulling statements removed in latest commit https://github.com/apache/cordova-plugin-inappbrowser/pull/362/commits/03deac68fb85a1454392d8aba4b717ff38d723f8